pyspecProcScripts/load_data.py

```diff
@@ -343,14 +71,19 @@ def proc_Hahn_echoph(s, fl=None):
     s.reorder("t", first=True)
     s.chunk("t", ["ph2", "ph1", "t2"], [2, 4, -1])
     s.labels({"ph2": r_[0.0, 2.0] / 4, "ph1": r_[0.0, 1.0, 2.0, 3.0] / 4})
+    s.set_prop("coherence_pathway", {"ph1": 1, "ph2": -2})
+    s.set_units("t2", "s")
+    s["t2"] -= s.get_prop("acq_params")["tau_us"] * 1e-6
+    s *= s.shape["nScans"]
+    s.squeeze()
     s.reorder(["ph2", "ph1"])
     s.setaxis("nScans", r_[0:nScans])
     s.reorder("t2", first=False)
     s.ft("t2", shift=True)
     if fl is not None:
         fl.next("raw data, chunked")
         fl.image(abs(s))
-    s.ft(["ph1", "ph2"], unitary=True)
+    s.ft(["ph1", "ph2"])
```

Comment:
explain the first time around that this will preseve the amplitude between individual transients and the signal pathway in the coherence domain.